collections-csharp/DictionaryGetValueByIndex/DictionaryTests/DictionaryGetValueByIndexTests.cs

```diff
@@ -0,0 +1,39 @@
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace DictionaryTests
+{
+    [TestClass]
+    public class DictionaryGetValueByIndexTests
+    {
+        [TestMethod]
+        public void GivenDictionary_WhenAccessedByIndex_ThenReturnElement()
+        {
+            var capitals = new Dictionary<string, string>(){
+                {"Turkey", "Ankara"},
+                {"UK", "London"},
+                {"USA", "Washington"}
+            };
+
+            var element = capitals.ElementAt(2);
+
+            Assert.AreEqual( "USA", element.Key);
```

Comment:
Formatting.

Reply:
What is the formatting problem, I can not get it.